lib/ansible/config/base.yml

```diff
@@ -913,6 +913,13 @@ DEFAULT_SCP_IF_SSH:
   env: [{name: ANSIBLE_SCP_IF_SSH}]
   ini:
   - {key: scp_if_ssh, section: ssh_connection}
+pkcs11_provider:
+  default: ""
+  description:
+    - "PKCS11 SmartCard provider such as opensc, example: /usr/local/lib/opensc-pkcs11.so"
+  env: [{name: ANSIBLE_PKCS11_PROVIDER}]
+  ini:
+    - {key: pkcs11_provider, section: ssh_connection}
```

Comment:
You don't need this, just the part in the connection plugin

Reply:
I removed this